sdks/python/apache_beam/coders/coder_impl.pxd

```diff
@@ -137,3 +138,6 @@ cdef class WindowedValueCoderImpl(StreamCoderImpl):
 
   @cython.locals(c=CoderImpl)
   cpdef get_estimated_size_and_observables(self, value, bint nested=?)
+
+  @cython.locals(wv=windowed_value.WindowedValue)
+  cpdef encode_to_stream(self, value, OutputStream stream, bint nested)
```

Comment:
We could. We'd have to specify it for the supertype. In general, however, I try to specify as little as needed.